ramls/inventory-move.raml

```diff
@@ -26,7 +26,7 @@ traits:
       application/json:
         type: items_move
     responses:
-      201:
+      200:
```

Comment:
Please update API versions.

Reply:
Current version is v0.1. What is expecting version - 1.0 or 0.2?

Reply:
I think `0.2`. It is a breaking change I assume, but API is still experimental (major version is `0`) so we need to update only minor version.